types/react-phone-number-input/index.d.ts

```diff
@@ -68,11 +75,15 @@ export interface CountrySelectComponentProps {
      * The list of all selectable countries (including "International")
      */
     options?: Array<{ value?: string; label: string; icon: React.Component }>;
-    tabIndex?: number | string;
+    tabIndex?: number;
     /**
      * The currently selected country code
      */
     value?: string;
+    /**
+     * Language translations
+     */
+    labels?: { [key: string]: string };
```

Comment:
From what I understand [labels](https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement/labels) is a read-only prop from a `<select/>` element. Is not a prop that can be passed!! 
Also, I looked in the [react-phone-number-input](https://gitlab.com/catamphetamine/react-phone-number-input) and did not see any usage in the `countrySelectComponent`.
What am I missing?

Reply:
In according to the documentation it is possible to pass a `labels` prop when you create a custom country `<select/>` https://gitlab.com/catamphetamine/react-phone-number-input/-/blob/master/README.md#L187.

Reply:
I am not sure that this example is the same as [countrySelectComponent](https://gitlab.com/catamphetamine/react-phone-number-input/-/tree/master#countryselectcomponent) passed to `<PhoneInput/>`.

Reply:
I don't use this package right now and don't sure if the `labels` prop is needed?
Did you have such a use case and typing was missing?
Maybe another interface should be used in this case?

Reply:
Here is a more explicit example. You can do something like that:

```typescript
import fr from 'react-phone-number-input/locale/fr.json';

const CountrySelect = ({
   value,
   onChange,
   labels,
   ...rest
}: CountrySelectComponentProps) => (
    <select
        {...rest}
        value={value}
        onChange={(event) => {
            if (onChange) {
                onChange(event.target.value || undefined);
            }
        }}
    >
        {getCountries().map((country: string) => (
            <option key={country} value={country}>
                {labels[country]} +{getCountryCallingCode(country)}
            </option>
        ))}
    </select>
);

CountrySelect.defaultProps = {
    labels: fr
};

<PhoneInput
        value={'+64271231234'}
        onChange={(value: string) => {
            console.log(value);
        }}
        countrySelectComponent={CountrySelect}
/>
```

With the `labels` props you can apply translations.

Reply:
Good, I would suggest also adding `labels` prop in `types/react-phone-number-input/react-phone-number-input-tests.tsx` as you did with `getCountries`.

Reply:
Added :)